src/api/tracking/withdrawals.ts

```diff
@@ -44,7 +51,7 @@ export async function listNurseryWithdrawals(
   if (data) {
     return data.map((datum) => {
       const { batchWithdrawals, ...remaining } = datum;
-      const speciesScientificNames = (batchWithdrawals as any[]).map(
+      const speciesScientificNames = ((batchWithdrawals || DELETED_SPECIES) as any[]).map(
```

Comment:
batchWithdrawals is undefined if species is deleted and there's only one withdrawal in the original batch